docs/design/FeatureSwitches.md

```diff
@@ -45,19 +47,26 @@ It is necessary to use this feature instead of the string-based bindings in Nati
 ### Example use-case
 
 String-based binding in code:
+
 ```c#
 label.BindingContext = new PageViewModel { Customer = new CustomerViewModel { Name = "John" } };
 label.SetBinding(Label.TextProperty, "Customer.Name");
 ```
 
 Compiled binding in code:
+
 ```csharp
 label.SetBinding<PageViewModel, string>(Label.TextProperty, static vm => vm.Customer.Name);
 // or with type inference:
 label.SetBinding(Label.TextProperty, static (PageViewModel vm) => vm.Customer.Name);
 ```
 
 Compiled binding in XAML:
+
 ```xml
 <Label Text="{Binding Customer.Name}" x:DataType="local:PageViewModel" />
 ```
+
+## MauiHotReloadSupport
+
+When disabled, hot reload will not work.
```

Comment:
The question I'm wondering when reading this, does this feature flag affect XAML hot reload or C# hot reload?

We already have the `$(UseInterpreter)` property, for example, that optionally brings in the mono component, `libmono-component-hot_reload`. It's for C# hot reload, but I guess this is for XAML hot reload?